src/bias/ReweightTemperaturePressure.cpp

```diff
@@ -239,8 +239,8 @@ ReweightTemperaturePressure::ReweightTemperaturePressure(const ActionOptions&ao)
 }
 
 double ReweightTemperaturePressure::getLogWeight() {
-  double energy=0;
-  double volume=0;
+  double energy=getArgument(0);
+  double volume=getArgument(1);
```

Comment:
what happen if you have more than one argument for energy or volume? Is this possible or there are checks to avoid it?

Reply:
Thanks @carlocamilloni . I just changed the lines you mentioned to:
```
  double energy=0.0; for(unsigned i=0; i<myenergy.size(); ++i) energy+=getArgument(i);
  double volume=0.0; for(unsigned i=0; i<myvol.size(); ++i) volume+=getArgument(myenergy.size()+i);
```
This should handle many inputs in ENERGY and VOLUME. If there is a cleaner way to do it, for instance using `myenergy[i]->get()` and `myvol[i]->get()` please let me know.